plugin-server/src/types.ts

```diff
@@ -392,9 +393,9 @@ export interface JobSpec {
 
 export interface Plugin {
     id: number
-    organization_id: string
+    organization_id: string | undefined
```

Comment:
Nit: I generally prefer the optional marker
```suggestion
    organization_id?: string
```